crates/ruff/src/rules/flake8_no_pep420/rules.rs

```diff
@@ -8,6 +8,23 @@ use crate::registry::Diagnostic;
 use crate::violation::Violation;
 
 define_violation!(
+    /// ### What it does
+    /// Checks for packages that are missing an `__init__.py` file.
+    ///
+    /// ### Why is this bad?
+    /// Python packages are directories that contain a file named `__init__.py`.
+    /// The existence of this file indicates that the directory is a Python
+    /// package, and so it can be imported the same way a module can be
+    /// imported.
+    ///
+    /// Directories that lack an `__init__.py` file can still be imported, but
+    /// they're indicative of a special kind of package, known as a namespace
+    /// package (see: [PEP 420](https://www.python.org/dev/peps/pep-0420/)).
+    ///
+    /// Namespace packages are a relatively new feature of Python, and they're
```

Comment:
>relatively new feature of Python

I think it would be nice to mention the Python version where they were introduced because "relatively new" doesn't really say anything since we don't know when this was written or what the author's perception of time is.

Reply:
Sure will fix. Thanks for reviewing these, I appreciate it.